GOVERNANCE.md

```diff
@@ -103,21 +183,25 @@ Steward is an additional privilege bestowed to 1 (or more) Contributors. The rol
 - Ability to decide on moderation decisions
 - Access to the `*@astro.build` email address
 
-**Nomination:**
+#### Nomination
 
 - Stewards cannot be self-nominated.
-- Only Core Contributors are eligible.
-- New stewards will be added based on a unanimous vote by the existing stewards.
-- In the event that someone is unreachable then the decision will be deferred. Discussion and approval will be done in private.
+- Only Core Maintainers are eligible.
+- New stewards will be added based on a unanimous vote by the existing steward(s).
+- In the event that someone is unreachable then the decision will be deferred.
+
+# Governance Playbook
 
 ## Voting
 
 Certain project decisions require a vote. These include:
 
-- Governance changes: simple majority (over 50%) conducted via GitHub PR approval.
-- Contributor membership (L2 and L3): discussion conducted via a temporary Discord channel open to qualified contributors for up to 3 days. Acceptance requires an overwhelming majority (over 70%) vote conducted by privately messaging a steward. Funneling both assenting and dissenting votes directly through stewards allows for anonymity when discussing the merits of a potential contributor.
+- Governance changes: simple majority (over 50%) vote conducted via GitHub PR approval.
```

Comment:
Simple majority of who? Maintainers? Core Maintainers?

Reply:
fixed